tests/postgres_tests/test_array.py

```diff
@@ -965,6 +965,37 @@ class Meta:
         form = Form({'field_0': '1', 'field_1': '2'}, instance=obj)
         self.assertFalse(form.has_changed())
 
+        obj = IntegerArrayModel(field=[1])
+        form = Form({'field_0': '', 'field_1': ''}, instance=obj)
+        self.assertTrue(form.has_changed())
+
+        field = SplitArrayField(forms.CharField(), required=False, size=2, remove_trailing_nulls=True)
+        self.assertIs(field.has_changed(['5'], [None]), True)
+        self.assertIs(field.has_changed(['a'], ['a', '']), False)
+        self.assertIs(field.has_changed(['url'], ['', 'url']), True)
+        self.assertIs(field.has_changed([None], ['url', None]), True)
+        self.assertIs(field.has_changed(['a'], ['b', None]), True)
```

Comment:
These assertions are not related with a bugfix, please move them to a separate commit.